templates/post.html

```diff
@@ -85,6 +85,81 @@
         </p>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-8">
+        <script>
+        function fieldValidate(field) {
+          /* call Mkto.setError(field, message) and return false to mark a field value invalid */
+          /* return 'skip' to bypass the built-in validations */
+          return true;
+        }
+        function getRequiredFieldMessage(domElement, label) {
+          return "This field is required";
+        }
+        function getTelephoneInvalidMessage(domElement, label) {
+          return "Please enter a valid telephone number";
+        }
+        function getEmailInvalidMessage(domElement, label) {
+          return "Please enter a valid email address";
+        }
+        </script>
+
+        <form class='p-form mktoForm mktoNoJS' action='https://pages.canonical.com/index.php/leadCapture/save' method='post'>
+          <h2>Sign up for email updates</h2>
+          <p>Choose the topics you're interested in</p>
+            <ul class="p-list">
+              <li class="p-list__item">
+              <div>
+                <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightscloudserver" name="insightscloudserver" value="yes" {% if post.topics[0].slug == 'cloud' or 'server'  %} checked="checked"{% endif %} />
```

Comment:
The `value` should be unique to the checkbox?

Reply:
Done